packages/block-library/src/query/variations.js

```diff
@@ -69,6 +69,15 @@ const variations = [
 				{},
 				[ [ 'core/post-title' ], [ 'core/post-date' ] ],
 			],
+			[
+				'core/query-pagination',
+				{},
```

Comment:
We don't need to pass the innerBlocks as they are defined as the default `template` in `QueryPagination` [here](https://github.com/WordPress/gutenberg/blob/trunk/packages/block-library/src/query-pagination/edit.js#L20).

Reply:
Thanks, I have removed them.